src/ScriptContext.tsx

```diff
@@ -19,9 +19,11 @@ enum SCRIPT_LOADING_STATE {
     RESOLVED = "resolved",
 }
 
+type ScriptLoadingState = `${SCRIPT_LOADING_STATE}`;
```

Comment:
@leogedler I was reading up on `enums` and I think we should get rid of it and replace it with the following:

```suggestion
const SCRIPT_LOADING_STATE = {
    INITIAL: "initial",
    PENDING: "pending",
    REJECTED: "rejected",
    RESOLVED: "resolved",
} as const;

type ScriptLoadingState = typeof SCRIPT_LOADING_STATE[keyof typeof SCRIPT_LOADING_STATE];
```

https://stackoverflow.com/a/60041791

Reply:
@gregjopa got it, make sense the explanation why is better use to use a const.

I have pushed the suggested change.